content/body/status.php

```diff
@@ -1,7 +1,6 @@
 <p>
-  Status messages allow screen readers and other assistive technology to tell users that content has changed on the page 
-  without interrupting what the user is doing by changing element has focused.  A perfect example of this 
-  use-case is in a search component like in the example below.
+  Status messages allow screen readers and other assistive technology to tell users that content has changed on the page. It does this without interrupting what the user is doing and changing which interactive element has focus.  
```

Comment:
I think this sentence can be improved like:

```suggestion
  Status messages allow screen readers and other assistive technology to tell users that content has changed on the page. It does this without interrupting what the user is doing by changing which interactive element has focus.  
```